UnrealThirdPersonMP/README.md

```diff
@@ -0,0 +1,100 @@
+# PlayFab Multiplayer Unreal Game Server Sample
+
+## Prerequisites
+- Download Unreal Engine Source Build by following [these instructions](https://docs.unrealengine.com/4.26/en-US/ProgrammingAndScripting/ProgrammingWithCPP/DownloadingSourceCode/) from the Unreal website
+- Download Unreal Engine (this was tested on 4.26.2)
+- Download the [Unreal PlayfabGSDK Plugin folder](https://github.com/PlayFab/gsdk/tree/master/UnrealPlugin)
+- Download Visual Studio (the [community version is free](https://visualstudio.microsoft.com/vs/community/))
+- When running locally you must have a copy of [Local Multiplayer Agent](https://github.com/PlayFab/MpsAgent/releases)
+- A developer.playfab.com account
+
+OPTIONAL:
+- Clone the [MpsSamples repository](https://github.com/PlayFab/MpsSamples)
```

Comment:
can you add a note that this is to test server allocation by using the MpsAllocatorSample?